_data/Asif/MohdAs1f.md

```diff
@@ -0,0 +1,5 @@
+name:Mohd Asif Khan 
+institution: Jaipur National University
+profile_pic: asif2.png 

```

Comment:
Incorrect file name, please do one of the following:
1. Change `asif2.png` to `asif (2).png` in the markdown file.
2. Rename `asif (2).png` to `asif.png` and change `asif2.png` to `asif.png` in the markdown file.
I'd suggest you rename the file!